radio/src/targets/horus/lcd_driver.cpp

```diff
@@ -142,6 +142,7 @@ static void LCD_NRSTConfig(void)
 }
 
 // TODO delay function
+#if !defined(RADIO_T18)
```

Comment:
What's that for ?

Reply:
Because that function is only used if that radio is not defined if you look down at line 158

Reply:
it would be nice to use delay_us anyhow!

Reply:
I left the original code mostly as-is and just fixed some compiler complaints.

Looking at the code that would be a huge number to put into delay_us, the person who put it there probably had a reason for it and sadly it is not commented either.

I'm guessing the datasheet of the display/controller would have the reason documented.